src/victorops.coffee

```diff
@@ -231,8 +235,8 @@ class VictorOps extends Adapter
     # remove keys from the victor ops incident keys list and from the brain
     # if they are older than 24 hours
     voIKeysFiltered = voIKeys.filter((item) ->
-      if new Date(item.timestamp).addDays(1) < new Date
-        @robot.brain.remove item.name
+      if new Date(item.timestamp).getDate() + 1 < new Date
+        #@robot.brain.remove item.name
```

Comment:
Since there are no more side-effects here, how would you feel about something along the lines of this:
`return (new Date(item.timestamp).getDate() + 1 >= new Date)`